bin/inasafe.py

```diff
@@ -15,28 +15,36 @@
         cli: command line interface
     .. versionadded:: 3.2
 """
-import shutil
 import logging
 import os
-import tempfile
 
-from PyQt4.QtCore import QSettings
-from docopt import docopt, DocoptExit
 from qgis.core import (
     QgsRasterLayer,
     QgsVectorLayer,
     QgsRectangle,
     QgsMapLayerRegistry,
     QgsCoordinateReferenceSystem)
+from PyQt4.QtCore import QSettings
 
+from docopt import docopt, DocoptExit
 
 from safe.test.utilities import get_qgis_app
 # make sure this line executes first
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
+from safe.common.version import get_version
 from safe.utilities.gis import qgis_version, validate_geo_array
 from safe.utilities.osm_downloader import download
+from safe.utilities.settings import setting
 from safe.impact_function.impact_function import ImpactFunction
+from safe.datastore.folder import Folder
+from safe.definitions.constants import PREPARE_SUCCESS, ANALYSIS_SUCCESS
+from safe.definitions.utilities import map_report_component
+from safe.definitions.reports.components import (
+    report_a4_blue,
+    standard_impact_report_metadata_pdf)
+from safe.report.impact_report import ImpactReport
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->